web/src/engine/osk/src/input/gestures/browser/oskSubKey.ts

```diff
@@ -24,12 +42,15 @@ export default class OSKSubKey extends OSKKey {
     let tKey = osk.getDefaultKeyObject();
```

Comment:
References:

https://github.com/keymanapp/keyman/blob/2366653818820e11645716c597e386a3d322ca7b/web/src/engine/osk/src/visualKeyboard.ts#L519-L529

and

https://github.com/keymanapp/keyman/blob/2366653818820e11645716c597e386a3d322ca7b/common/web/keyboard-processor/src/keyboards/activeLayout.ts#L32-L38